core/src/test/scala/stryker4s/config/TestFilterTest.scala

```diff
@@ -0,0 +1,32 @@
+package stryker4s.config
+
+import stryker4s.testutil.Stryker4sSuite
+
+class TestFilterTest extends Stryker4sSuite {
+  describe("filter with default") {
+    implicit val conf: Config = Config.default
+    it("should work") {
```

Comment:
The 'should work' is a little duplicate three times. Perhaps a single `describe` of the function you are testing, with an `it` of the changed system under test description would be clearer?

Reply:
I fixed this